webapp/lib/app/nook/view.dart

```diff
@@ -1243,7 +1243,20 @@ class ConversationFilter {
     conversationFilter.append(_tagsContainer);
 
     _tagAccordion = Accordion([]);
-    _tagsMenuWrapper.append(_tagAccordion.renderElement);
+    var accordionActions = DivElement()..classes.add('filter-accordion-actions');
+    var expandButton = buttons.Button(buttons.ButtonType.text, buttonText: "Expand all", onClick: (e) {
+      _tagAccordion.items.forEach((AccordionItem item) { item.expand(); });
```

Comment:
nit: 
```suggestion
      _tagAccordion.items.forEach((AccordionItem item) => item.expand());
```
(this is the dart syntax for one liner functions)